homeassistant/components/stookwijzer/sensor.py

```diff
@@ -66,6 +76,15 @@ async def async_setup_entry(
         StookwijzerSensor(description, entry) for description in STOOKWIJZER_SENSORS
     )
 
+    platform = entity_platform.async_get_current_platform()
+
+    platform.async_register_entity_service(
+        "get_forecast",
+        None,
+        "async_get_forecast",
+        supports_response=SupportsResponse.ONLY,
+    )
```

Comment:
Why is this a platform service and not a normal service that aims to a config entry? It doesn't seem like that the entity you pick has any significance on what is returned, so IMO we can just make this a service on integration level, rather than entity level

Reply:
Check Mealie or Overseerr for an example for this

Reply:
Very good suggestion. I've changed this, and the tests also.